src/app/static/src/app/components/header/DataExplorationLanguageMenu.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <language-menu-view :change-language="changeLanguage"
+                        :current-language="currentLanguage">
+    </language-menu-view>
+</template>
+<script lang="ts">
+
+    import Vue from "vue";
+    import {mapActionByName, mapStateProp} from "../../utils";
+    import {DataExplorationState} from "../../store/dataExploration/dataExploration";
+    import LanguageMenuView from "./LanguageMenuView.vue";
+
+    interface Methods {
+        changeLanguage: (lang: string) => void
+    }
+
+    interface Computed {
+        currentLanguage: string
+    }
+
+    export default Vue.extend<unknown, Methods, Computed, unknown>({
+        computed: {
+            currentLanguage: mapStateProp<DataExplorationState, string>(null,
+                (state: DataExplorationState) => state.language.toUpperCase())
```

Comment:
So the only difference between this and the `LanguageMenu.vue` component is that this references `DataExplorationState` instead of `RootState`. But `RootState` inherits from `DataExplorationState`, so it would be fine to just use the same component for both. Because both Vuex stores now have an action named "changeLanguage" this component will work just fine for both apps!